rules/coding-style/src/Rector/Throw_/AnnotateThrowablesRector.php

```diff
@@ -121,6 +121,11 @@ public function throwException(int $code)
         );
     }
 
+    /**
+     * @param Node|Throw_|MethodCall|FuncCall $node
+     *
+     * @return Node|null
```

Comment:
Duplicated information, can be removed

Reply:
Yes, it is not checked with style fixes... It is just to make you able to read the new code...